cypress/intercepts/CreateAdmin.ts

```diff
@@ -0,0 +1,20 @@
+export const createAdmin = () => {
+    cy.intercept("POST", "/admin", {
+      statusCode: 200,
+      body: {
+        id_account: 1,
+        email: "",
+        account_category: "admin",
+      },
+    });
+  };
+
+  export const meSuperadmin = () =>
+  cy.intercept("GET", "/me", {
+    statusCode: 200,
+    body: {
+      id_account: 1,
+      email: "superadmin@kosjenka.com",
+      account_category: "superadmin",
+    },
+  });
```

Comment:
This one already exists under login folder, please remove this one and use the other instead.